README.md

```diff
@@ -6,7 +6,7 @@ This repo includes guides to get started with Kafka client libraries in the foll
 - [C# / .NET](dotnet/README.md)
 - [Go](go/README.md)
 - [Java](java/README.md)
-- [Node.js](nodejs/README.md)
+- [JavaScript](nodejs/README.md)
```

Comment:
should the folder be renamed as well? or will it break tests etc

Reply:
discussed IRT and settled on doing the folder / page slug rename later as a follow-on once redirects are fixed on CD.